presto-spi/src/main/java/io/prestosql/spi/procedure/Procedure.java

```diff
@@ -92,11 +110,25 @@ public String toString()
     {
         private final String name;
         private final Type type;
+        private final boolean isRequired;
```

Comment:
The `is` prefix is typically reserved for getters (like `get` but for booleans), not used for fields or parameters, so name this just `required`.